### DISCUSSION TITLE
VIMAR LINEA problemi con zigbee2mqtt

### DISCUSSION THREAD

**Discussion by federicotravaini** [indomus/forum#3319] [ZigBee, Z-Wave, Thread, Matter]
Premetto che sono settimane che sto perdendoci la testa e a questo punto spero nella saggezza divina di qualcuno di voi.

Casa nuova, impianto luce con deviatori connessi VIMAR LINEA. 
Coordinator zigbee SLBZ-06. 
Software: Z2M

I deviatori non ne vogliono sapere di funzionare come dovrebbero. Si collegano, poi si scollegano. Funzionano un po' si e un po' no. Ho provato un po' di tutto, ho cambiato posizione al coordinator, ho provato diversi firmware, diverse release di Z2M ma non trovo un minimo di stabilità.

Il resto della rete è composto da termostati MOES e deviatori BTIcino living now light e funziona tutto discretamente.

Ho fatto anche un mezzo tentativo usando ZHA ma lo trovo troppo semplice e ho paura che a un certo punto potrei sentire l'esigenza di tornare a Z2M.

Quasi dimenticavo: rete zigbee su canale 11, mentre wifi sono su canale 6 e 11

**Comment by goolasso** [comment]
Anche io ho un problema simile. A casa da circa 6 mesi sto utilizzando HA su RPI, antenna sonoff e z2m. Ho installato e configurato circa 15 device Vimar oltre ad altri oggetti terze parti. Uso anche google home attraverso cloudflare. Ho un problema piuttosto fastidioso: ogni volta che va via la corrente perdo diversi device collegati e devo fare un re-paring o a volte anche un reset per associarli di nuovo, e questo mi crea anche altri problemi vari al contorno. A volte mi sembra di perdere alcuni device anche senza che va via la corrente, ma non ne sono sicuro. Quindi sostanzialmente il tutto è piuttosto inaffidabile, però probabilmente sto sbagliando qualcosa...tu hai trovato una qualche soluzione?

**Reply by federicotravaini** [reply]
purtroppo ho "risolto" comprando il gateway e configurando tutti i deviatori con BT per poi connetterli ad HA usando HomeKit

**Reply by seattle93** [reply]
Ciao Federico , potresti spiegarmi la procedura ? Ho la serie VIMAR connessa però non riesco a integrare tapparelle e luci su home assistant. Grazie

**Reply by federicotravaini** [reply]
ho acquistato il gateway VIMAR, connesso tutti i deviatori con l'app VIMAR, e poi ho integrato il gateway in Home Assistant tramite HomeKit

**Reply by seattle93** [reply]
Ho risolto grazie.
Il problema è che l’avevo associato anche su HomeKit , l’ho dissociato poi ho fatto un sincronizza su vimar VIEW wireless del gateway e infine ho provato ad associarlo su HomeKit 👌

**Comment by misterbit74** [comment]
Il cloud e i device radio che non si collegano piu' mi fanno venire l'orticaria....
Se possibile fate il piu' possibile via cavo e in locale.

**Comment by saveriol** [comment]
Anche io ho avuto diversi problemi con la linea Vimar VIEW (immagino intendessi quella): ho aperto/collaborato nel tempo anche diverse issue in Z2M

https://github.com/Koenkk/zigbee2mqtt/issues/26945
https://github.com/Koenkk/zigbee2mqtt/issues/20045
https://github.com/Koenkk/zigbee2mqtt/issues/26676

da quello che ho potuto capire, Vimar non è pienamente aderente allo standard Zigbee. Ad esempio i device cambiano a caso e in autonomia il network address, quando non dovrebbero. Per questo gli sviluppatori di Z2M hanno messo una pezza, andando a chiedere i network address aggiornati alla rete se ricevono un pacchetto da un device sconosciuto (perché ha cambiato indirizzo).
VIMAR non sembra essere propensa a migliorare i suoi firmware per gateway che non siano quelli di SmartThings/Alexa, li ho contattati più volte ma non ho ottenuto risposte...
[L'ultimo dispositivo](https://www.vimar.com/en/int/catalog/product/index/code/30801) che hanno rilasciato in zigbee semplicemente va offline a caso...
